Code/Tools/CrashHandler/Platform/Android/PAL_android.cmake

```diff
@@ -6,4 +6,3 @@
 #
 #
 
-set(PAL_TRAIT_BUILD_CRASH_HANDLER_SUPPORTED FALSE)
```

Comment:
Removed from the module PAL as definition is not properly applied (EXTERNAL_CRASH_REPORTING ifdef, defined by the module when this pal is on, is disabled if it is defined there instead of in the global o3de pal cmake)